### ISSUE TITLE
Light theme has too low contrast on nickname coloring

### ISSUE THREAD

**Issue by fuzzy76** [glowing-bear/glowing-bear#534]
The nickname colors are so similar, it's almost no use.

**Comment by Evropi** [comment]
Indeed. The light theme in general needs typographical and design improvements. These go beyond the 16-colour set.

For example, the background is currently #F7F7F7, the purest shade of white it can be. But it could be #EEE or a shade of grey and would probably look rather better.

**Comment by torhve** [comment]
None of the developers are designers, so we would likely welcome any attempt at improving the design. I suggest playing around with the design and taking screenshots before and after. Then we can discuss the changes here or on IRC and get them into the theme if we agree on improvements.

**Comment by cormier** [comment]
Exactly what @torhve said. I have added the light theme to make it easier for people to contribute.
Everything should be in [light.css](https://github.com/glowing-bear/glowing-bear/blob/master/css/themes/light.css), so feel free to play with the colors and submit a PR!

**Comment by txomon** [comment]
BTW, I would also add the fact about joins/parts. Makes it difficult to distinguish between text and those...
